### PR TITLE
🔥 Use laravel own encryptable casting

### PR DESCRIPTION
**Pull request by MrKrisKrisu**

### CODE REVIEW

**Comment by MrKrisKrisu** [comment]
@HerrLevin  Please review this, but **DO NOT MERGE** it. I would like to do the release preferably at night to cause little conflict.

File: database/migrations/2021_11_03_232436_migrate_encryptables.php
Comment (HerrLevin) [inline database/migrations/2021_11_03_232436_migrate_encryptables.php]: I'm not entirely sure if this will work while the attribute is already casted as a laravel encryptable.

**Review by HerrLevin — COMMENTED** [review]
The code looks fine to me but **PLEASE** test it thoroughly.

File: database/migrations/2021_11_03_232436_migrate_encryptables.php
Comment (MrKrisKrisu) [reply database/migrations/2021_11_03_232436_migrate_encryptables.php]: The encryptable trait is removed from me and ->getAttributes() will return it raw.

**Review by HerrLevin — COMMENTED** [review]
The code looks fine to me but I'll need to test it before I can approve it

**Comment by MrKrisKrisu** [comment]
I've tested it locally.

**Review by HerrLevin — APPROVED** [review]

**Comment by MrKrisKrisu** [comment]
Thanks @xanderio!

**Pull request merged**
